maps/randomvaults/objects.dm

```diff
@@ -939,3 +939,37 @@
 		/obj/item/weapon/stock_parts/console_screen\
 	)
 
+
+/area/vault/angies_lair
+	name = "asteroid internals"
+	flags = NO_PACIFICATION|NO_PERSISTENCE|NO_MESONS|NO_TELEPORT|NO_PORTALS
+
+/turf/unsimulated/mineral/destroy_surrounding/GetDrilled(var/artifact_fail = FALSE, var/safety_override = FALSE, var/atom/driller)
+	..()
+	spawn(1 SECONDS)
+	for(var/dir in cardinal)
```

Comment:
You meant to indent this with one more tab, though I don't know why it's spawn()ing at all.

Reply:
I used a sleep previously, and it would end up in a chain of each destroy surroundings waiting for the next one to destroy its surroundings, which would be waiting for the next one to destroy its surroundings..